tests/test_enforcer.py

```diff
@@ -25,7 +25,7 @@ def get_examples(path):
     return os.path.abspath(examples_path + path)
 
 
-class TestSub:
+class MockSub:
```

Comment:
this was triggering warnings, since pytest et al assume things beginning with `Test` are actual tests